ambry-router/src/main/java/com/github/ambry/router/PutBlobMetaInfo.java

```diff
@@ -45,32 +46,63 @@
  *             "size": 104
  *         }
  *     ],
- *     "reservedMetadataChunkId": "AAYIAkJbXNUAAQAAAAAAAAACEEbJqS49Sz2YIfqoDrpHmw"
+ *     "version": 1
```

Comment:
BTW, if we are adding versions to etag, then we should add version outside of the serialized PutBlobMetaInfo, so later if we decide not to use PutBlobMetaInfo as etag, we can use the version number to tell.

For example, the etag should be v1:{"blob": {}}, notice that v1 is outside of the serialized json string. Later if we decide to not use PutBlobMetaInof, we can do v2:xxxxxx

What do you think?